src/openrct2-ui/windows/Window.h

```diff
@@ -9,12 +9,14 @@
 
 #pragma once
 
+#include <openrct2-ui/input/KeyboardShortcuts.h>
 #include <openrct2-ui/interface/Window.h>
 #include <openrct2/common.h>
 #include <openrct2/ride/Ride.h>
 #include <openrct2/windows/tile_inspector.h>
 #include <string_view>
 
+using namespace OpenRCT2;
```

Comment:
Don't add using namespace in a popular header like this your bringing the whole of the namespace in when you do not need to. Completely removing the point of namespaces.